lib/train/extras/command_wrapper.rb

```diff
@@ -53,9 +53,27 @@ def initialize(backend, options)
       @user          = options[:user]
     end
 
+    def with_sudo_pty
+      old_pty = backend.transport_options[:pty]
+      backend.transport_options[:pty] = true if @sudo
+
+      yield
+    ensure
+      backend.transport_options[:pty] = old_pty
+    end
+
     # (see CommandWrapperBase::verify)
     def verify
-      res = @backend.run_command(run("echo"))
+      cmd = if @sudo then
+              # TODO: double bash -c wrapper... I need /dev/null on the outside
+              "bash -c '(#{run("-v")}) < /dev/null'"
```

Comment:
This line might benefit from an explanatory code comment. Especially when working in non-Ruby languages...